epistasis_selection.py

```diff
@@ -22,44 +22,31 @@
 import utils
 
 
-def combine_k_rounds(num_rounds: int, mutations: Iterable[Sequence[Tuple[int, int]]]) -> List[Sequence[Tuple[int, int]]]:
+def combine_k_rounds(num_rounds: int, mutations: Iterable[Tuple[Tuple[int, int], ...]]) -> List[Tuple[Tuple[int, int], ...]]:
```

Comment:
thanks for making this change. I forgot about this comment when I lgtm'd the previous PR. This is much cleaner.